coil-gif/src/main/java/coil/drawable/MovieDrawable.kt

```diff
@@ -38,6 +40,8 @@ class MovieDrawable @JvmOverloads constructor(
 
     private val callbacks = mutableListOf<Animatable2Compat.AnimationCallback>()
 
+    private val size = PixelSize(movie.width(), movie.height())
```

Comment:
This will crash if either of `movie`'s dimensions are -1 which is common. We can safely remove this once `AnimatedTransformation` is updated.

Reply:
Done